src/utils/hammer.js

```diff
@@ -18,19 +18,4 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// Hammer.Manager mock for use in environments without `document` / `window`.
-function HammerManagerMock(m) {
-  const instance = {};
-  const chainedNoop = () => instance;
-  instance.get = () => null;
-  instance.set = chainedNoop;
-  instance.on = chainedNoop;
-  instance.off = chainedNoop;
-  instance.destroy = chainedNoop;
-  instance.emit = chainedNoop;
-  return instance;
-}
-
-export const Manager = HammerManagerMock;
-
 export default null;
```

Comment:
I guess unrelated to this diff, but why explicitly export default null?

Reply:
This was originally an attempt to separate Node-only code and browser-only code. Now that the browser part does not work in all cases it does seem redundant. :(